pkg/controllers/update.go

```diff
@@ -252,7 +252,12 @@ func InstallUpdate(version string) {
 	var controllerErr Error
 	if utils.IsWindows() && !utils.IsMINGW64() {
 		if installedViaWinget() {
-			wasUpdated, installedVersion, controllerErr = updateViaWinget(version)
+			if err := updateViaWinget(version); err != nil {
+				utils.HandleError(err, "Unable to execute winget")
+			}
+
+			utils.LogDebug("Executing winget in background, CLI is now exiting")
+			os.Exit(0)
```

Comment:
check: Now that we're exiting immediately, we're no longer calling `SetVersionCheck` at the bottom of this function. Does this create any problems?